spec/index.bs

```diff
@@ -281,18 +281,20 @@ interface GPUShaderStageBit {
 
 enum GPUBindingType {
     "uniform-buffer",
-    "dynamic-uniform-buffer",
-    "sampler",
-    "sampled-texture",
     "storage-buffer",
-    "dynamic-storage-buffer"
+    "readonly-storage-buffer",
+    "sampler",
+    "sampled-texture"
     // TODO other binding types
 };
 
 dictionary GPUBindGroupLayoutBinding {
     required u32 binding;
     required GPUShaderStageFlags visibility;
     required GPUBindingType type;
+    // For uniform, storage and readonly storage buffer, means that the binding
+    // has a dynamic offset.
```

Comment:
input from meeting: PR looks good, but explain what the validation difference is between dynamic=true and dynamic=false. (does dynamic=false mean offsets have to be 0, or that they have to not be used?)

Reply:
Done